src/Module/Api/Mastodon/Statuses.php

```diff
@@ -132,6 +132,9 @@ public function put(array $request = [])
 			}
 			unset($item['attachments']);
 		}
+		if (!Item::isValid($item)) {
+			throw new \Exception('Missing parameters in definitien');
```

Comment:
Beyond the typo, what does this mean? What's the remediation?
```suggestion
			throw new \Exception('Missing parameters in definition');
```

Reply:
This line was added via commit https://github.com/friendica/friendica/commit/7ccc978bc5c51b784af8b94ffb1e4da08849cc29